wgsl/index.bs

```diff
@@ -1693,9 +1693,13 @@ Unless explicitly permitted below, an attribute [=shader-creation error|must not
     Each parameter [=shader-creation error|must=] be a [=const-expression=] or an [=override-expression=].
     All parameters [=shader-creation error|must=] be the same type, either [=i32=] or [=u32=].
 
-    A [=pipeline-creation error=] results if any specified parameter is 0 or exceeds an upper bound
-    specified by the WebGPU API, or if the product of the parameter values exceeds the
-    upper bound specified by the WebGPU API (see [[WebGPU#limits]]).
+    A [=shader-creation error=] results if any specified parameter is a
+    [=const-expression=] that evaluates to a non-positive value.
```

Comment:
This looks like a potentially breaking change by moving some errors from pipeline creation to shader creation (preventing creation of pipelines from other entry points in the same shader module). Do we know what chrome's current behavior is?

Reply:
Tint rejects shaders that use a constant workgroup size value less than 1, so Chrome already produces a shader creation error in this case.